docs/DeveloperGuide.md

```diff
@@ -1,34 +1,247 @@
 # Developer Guide
 
-## Design & Implementation
+## 1. Setting Up
+   * Prerequisites
+     * JDK 11 or above
+     * IntelliJ IDE
+   * Setting up the project in your computer
+     * Fork this repo, and clone the fork to your computer
+     * Open IntelliJ (if you are not in the welcome screen, click File > Close Project to close the existing project dialog first)
+     * Set up the correct JDK version for Gradle
+     * Click Configure > Project Defaults > Project Structure
+     * Click New…​ and find the directory of the JDK
+     * Click Import Project
+     * Locate the build.gradle file and select it. Click OK
+     * Click Open as Project
+     * Click OK to accept the default settings.
+   * Verifying the setup
+     * Run the jikan.jikan and try a few commands
+     * Run the tests and ensure they all pass.
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## 2. Design
+![image_info](./pictures/ClassDiagram.png)
```

Comment:
The UML diagram describing the design of the program is too complicated. Perhaps it would be better to break the design down into smaller components to describe? 
Also, your team could consider adding some description in words to the diagram so that it would be easier for the reader to understand the diagram?